### PR TITLE
Task/268 create a default application config

### PR DESCRIPTION
**Pull request by mirekgab**
#### Checklist template for LEADERS

&#x2611; | &#x2610;

&#x2610; New code is covered with tests and coverage requirements are met.
&#x2610; Build is not failing.
&#x2610; All tests are passing.
&#x2610; All conflicts are resolved and the branch is rebased on top of the up-to-date parent branch.
&#x2610; All issue requirements are implemented.
&#x2610; At least two reviewers are assigned.
&#x2610; The Pull request is linked to the related issue.
&#x2610; Correct branch name. Convention [https://github.com/bycza-zagroda/track-expenses-app-documentation/blob/develop/githubwork/GITHUB_WORK.md]
&#x2610; Correct unit test name - suffix 'Test' in the end.
&#x2610; Correct integration test name - suffix 'IT' in the end.
&#x2610; Positive and negative test paths have been implemented.

### CODE REVIEW

**Review by poneciak57 — DISMISSED** [review]
Looks good to me

File: src/main/resources/application-dev.yaml [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
 spring:
-  banner:
-    location: classpath:banner.txt
   datasource:
     url: jdbc:mysql://${DB_SERVER}:${DB_PORT}/${DB_NAME}?useUnicode=true&serverTimezone=UTC
     username: ${DB_USER}
     password: ${DB_PASSWORD}
-    driver-class-name: com.mysql.cj.jdbc.Driver
 
 #to zeby wyrzucaly sie castomowe wyjatki gdy nie ma endointu jeszcze
```
Comment (dawciobiel) [inline src/main/resources/application-dev.yaml:8]: It seems to me that it would be better to use comments in English in the code.

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+springdoc:
+  api-docs:
+    path: /docs/api-docs
+    enabled: true
+  swagger-ui:
+    path: /docs/swagger
+    enabled: true
+
+jwt:
+  secret: Blabla
```
Comment (dawciobiel) [inline src/main/resources/application.yaml:34]: Why does the secret have the value "blabla" instead of the part of a very easy to type, remember and in general obvious obviousness such as [ "password" | "secret" | "admin1" | "Ja ro sław Pol skę zbaw" ] ?

**Review by dawciobiel — CHANGES_REQUESTED** [review]
Please let me know about my comments. Other than that it looks good to me.

File: src/main/resources/application-test.yaml [hunk 3]
```diff
@@ -1,45 +1,16 @@
...
   liquibase:
     change-log: db/dbchangelog-test.yaml
     drop-first: true
-  #Jpa/Hibernate:
-  jpa:
-    open-in-view: false
-    generate-ddl: false
-    hibernate:
-      ddl-auto: none
-
```
Comment (Iwona007) [inline src/main/resources/application-test.yaml:30]: But here we have different hibernate strategy. Should it stay here, shouldn't?

File: src/main/resources/application-test.yaml [hunk 4]
```diff
@@ -1,45 +1,16 @@
...
-  banner:
-    location: classpath:banner.txt
   datasource:
     url: jdbc:mysql://localhost:3306/trackexpensesapptest?useUnicode=true&serverTimezone=UTC
     username: root
     password: root
-    driver-class-name: com.mysql.cj.jdbc.Driver
-    hikari:
-      connection-timeout: 500
-      maximum-pool-size: 5
```
Comment (Iwona007) [inline src/main/resources/application-test.yaml:7]: Why did you delete pool-size and timeout from test profile?

File: src/main/resources/application-dev.yaml [hunk 5]
```diff
@@ -23,31 +16,9 @@ spring:
...
-    generate-ddl: false
-    hibernate:
-      ddl-auto: validate
-    database-platform: org.hibernate.dialect.MySQL8Dialect
-    #do debugowania sql
-    show-sql: true
-    properties:
-      hibernate:
-        format_sql: true
-
```
Comment (Iwona007) [inline src/main/resources/application-dev.yaml:38]: 34-37 Lines was was only in dev profile.  Do we need them in general profiles?

**Review by Iwona007 — CHANGES_REQUESTED** [review]
Request changes or explanation

File: src/main/resources/application-test.yaml (hunk 4 above)
Comment (mirekgab) [reply src/main/resources/application-test.yaml:7]: This setting was added to fix an issue with test containers that no longer exists after recent changes to the testcontainers configuration.

File: src/main/resources/application-dev.yaml (hunk 5 above)
Comment (mirekgab) [reply src/main/resources/application-dev.yaml:38]: You're right, I'll correct it.

File: src/main/resources/application-test.yaml (hunk 3 above)
Comment (mirekgab) [reply src/main/resources/application-test.yaml:30]: I think validation of the database schema with entities should take place in every profile. Correct me if I'm wrong.

File: src/main/resources/application-dev.yaml (hunk 1 above)
Comment (mirekgab) [reply src/main/resources/application-dev.yaml:8]: I removed the unnecessary comment

File: src/main/resources/application.yaml (hunk 2 above)
Comment (mirekgab) [reply src/main/resources/application.yaml:34]: This value was set during development and I haven't changed it. I think we should use an environment variable to set this value before release. I'm adding a comment so we can remember this.

File: src/main/resources/application-test.yaml (hunk 3 above)
Comment (Iwona007) [reply src/main/resources/application-test.yaml:30]: I don't think so, because integration tests have cleaning schema. So for every test case database will be created again.

**Review by Iwona007 — APPROVED** [review]

**Review by dawciobiel — APPROVED** [review]
Looks good to me
